rtgui/previewhandler.cc

```diff
@@ -204,27 +204,19 @@ Glib::RefPtr<Gdk::Pixbuf> PreviewHandler::getRoughImage (int x, int y, int w, in
             h = image->getHeight() * totalZoom;
         }
 
-        int ix = x * zoom;
-        int iy = y * zoom;
+        x *= zoom;
+        y *= zoom;
 
-        if (ix < 0) {
```

Comment:
No, we don't need to check that anymore. `x` and `y` are `cropX` an `cropY` from the caller which are obtained through `cropHandler.getPosition`. The returned values (by reference) of this function can't be negative (`CropHandler::cropX` and `CropHandler::cropY` are trimmed in `CropHandler::compDim`).